EssentialsDiscord/src/main/resources/config.yml

```
@@ -299,6 +305,17 @@ messages:
   # - {displayname}: The display name of the user who is no longer afk
   # ... PlaceholderAPI placeholders are also supported here too!
   un-afk: ":keyboard: {displayname} is no longer AFK!"
+  # This is the message sent to discord when a player is awarded an advancement.
+  # The following placeholders can be used here:
+  # - {username}: The name of the user who was awarded the advancement.
+  # - {displayname}: The display name of the user who was awarded the advancement.
+  # - {advancement}: The name of the advancement.
+  # ... PlaceholderAPI placeholders are also supported here too!
+  advancement: ":medal: {displayname} has completed the advancement **{advancement}**!"
+  # This is the message sent to discord when the server starts.
```

```suggestion
  # This is the message sent to Discord when the server starts.
```

I realise this isn't new but it slipped through previously

Also, why move these? Will introduce confusion for people updating configs, and I don't see a reason to move these all the way down here?

---

Mistake from before, I want player-specific (and thus PAPI supporting) message types on the top, others on bottom. Not too worried about it, we haven't done a full release yet.